docs/source/conf.py

```diff
@@ -118,7 +118,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
```

Comment:
When building the docs with `make html` (as described in [the docs](https://jupyter-notebook.readthedocs.org/en/latest/contributing.html#building-the-documentation)), the built files end up in `docs/build/` and not in `docs/source/_build`, so I don't think it's necessary to have latter in the `exclude_patterns`.

But if you want it there anyway, just say the word, and I'll add it back!